components/packaging/common/load-telemetry-plugin

```diff
@@ -0,0 +1,66 @@
+#!/usr/bin/env sh
+
+BASE=/var/lib/docker
+TAR_DIR=$BASE/plugins/tar
+
+# make sure that the directory exists
+mkdir -p $TAR_DIR
+
+# Create the engine UUID, if it doesn't exist
+# This will be eventually handled by the docker engine
+# ref. https://github.com/moby/moby/issues/33356
+if [ -f $BASE/engine_uuid ]; then
+    UUID=$(cat $BASE/engine_uuid)
+fi
+if [ -z ${UUID+x} ]; then
+    UUID=$(cat /proc/sys/kernel/random/uuid | tee $BASE/engine_uuid)
+fi
+
+# make sure that docker is running
+if ! printf "%s" "$DOCKER_OPTS" | grep -qE -e '-H|--host'; then
```

Comment:
`upstart` an `sysvinit` are the only ones using that. It's "safe" to use in those, because they _know_ they're upstart/sysvinit, but this script is run on all init systems, including `systemd`